distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,53 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+
+
+
+=== Configuration Export/Import
+==== Saving Configurations
```

Comment:
Let's see what @ricklarsen has to say about this section. Not sure we want to be the ones explaining this here.